pkg/cloudinit/cloudconfig.go

```diff
@@ -18,9 +18,11 @@ import (
 )
 
 type CloudConfig struct {
-	Apt            Apt                `yaml:"apt,omitempty"`
-	Packages       []string           `yaml:"packages,omitempty"`
-	Runcmd         [][]string         `yaml:"runcmd,omitempty"`
+	Apt           Apt      `yaml:"apt,omitempty"`
+	Packages      []string `yaml:"packages,omitempty"`
+	PackageUpdate bool     `yaml:"package_update,omitempty"`
+	// Runcmd could be either []string or string
```

Comment:
runcmd results with []string and string are not the same. One was run with exec, One with sh. There was once a need for sh run during development of this example. It's no longer needed. But however it doesn't hurt to support both of them.